client/network-gossip/src/bridge.rs

```diff
@@ -177,23 +227,34 @@ impl<B: BlockT> Future for GossipEngine<B> {
 
 #[cfg(test)]
 mod tests {
-	use super::*;
+	use async_std::task::spawn;
 	use crate::{ValidationResult, ValidatorContext};
+	use futures::{channel::mpsc::{channel, Sender}, executor::block_on_stream};
+	use sc_network::ObservedRole;
+	use sp_runtime::{testing::H256, traits::{Block as BlockT, Hash}};
```

Comment:
```suggestion
	use sp_runtime::{testing::H256, traits::{Block as BlockT}};
```

Tests fail because of this unused import..